docs/source/configuration.rst

```diff
@@ -57,23 +57,36 @@ Your server/docker container clock should also match the accurate time of day!
     navigation bar! All other timestamps are in the same time zone!
 
 
-Using Amazon SES instead of SMTP email
---------------------------------------
```

Comment:
keep the old headline + section intact b/c it results in a bookmarkable link. Instead add the SMTP configuration section before it.